### PR TITLE
Use Package suffix instead of PackageId and Version suffix instead of PackageVersion

### PR DESCRIPTION
**Pull request by ViktorHofer**
I'm consolidating all the version properties to use the `Version` suffix instead of `PackageVersion`. This aligns to Arcade conventions and extension point which also use the `Version` suffix.

Requires https://github.com/dotnet/arcade/pull/4266

### CODE REVIEW

**Comment by ViktorHofer** [comment]
I will update this PR as soon as an Arcade build is available.

**Comment by ViktorHofer** [comment]
@maryamariyan @danmosemsft In cases where we do changes like this one, tagging every area-owner doesn't make much sense. I already requested a review from @safern here. Especially in the new runtime repo where we will have area-owners from the different sub-repos.

**Comment by maryamariyan** [comment]
@ViktorHofer I just ran a one-time script to request review from area owners for each of the open PRs.

So just to confirm, your recommendation is for future, not to assign the rest of the owners if one has already been requested?

**Comment by danmosemsft** [comment]
We should probably do a bit more thinking about how to best tag owners so it's useful without making lots of noise as well. We could wait til after hte repo merge I think

**Comment by ViktorHofer** [comment]
Of course this can wait. Just wanna share my opinion in case you want to do this more regularly.

File: eng/Versions.props [hunk 1]
```diff
@@ -22,63 +22,60 @@
...
-    <TraceEventPackageVersion>2.0.5</TraceEventPackageVersion>
-    <NewtonsoftJsonPackageVersion>12.0.1</NewtonsoftJsonPackageVersion>
-    <MicrosoftDotNetPlatformAbstractionsPackageVersion>3.0.0-preview6-27804-01</MicrosoftDotNetPlatformAbstractionsPackageVersion>
+    <MicrosoftNETTestSdkVersion>16.3.0</MicrosoftNETTestSdkVersion>
+    <XUnitVersion>2.4.1</XUnitVersion>
+    <TraceEventVersion>2.0.5</TraceEventVersion>
+    <NewtonsoftJsonVersion>12.0.1</NewtonsoftJsonVersion>
+    <MicrosoftDotNetPlatformAbstractionsVersion>3.0.0-preview6-27804-01</MicrosoftDotNetPlatformAbstractionsVersion>
     <!-- Test data -->
     <SystemIOCompressionTestDataPackageVersion>1.0.16</SystemIOCompressionTestDataPackageVersion>
```
Comment (safern) [inline eng/Versions.props]: You're missing this property.

**Review by safern — APPROVED** [review]
Other than one comment LGTM

File: eng/Versions.props (hunk 1 above)
Comment (ViktorHofer) [reply eng/Versions.props]: thanks

**Comment by ViktorHofer** [comment]
Merging as the build is completed.

**Pull request merged**